internal/control_plane/id_generator/idgenerator.go

```diff
@@ -0,0 +1,15 @@
+package id_generator
+
+import (
+	id "github.com/sony/sonyflake"
+	"octavius/internal/control_plane/logger"
+)
+
+func NextID() uint64 {
```

Comment:
please add an error to the return. and add a comment in exported function.

Reply:
added comment to the exported function